src/tokens/theme.native.js

```diff
@@ -0,0 +1,27 @@
+import { fonts, colors } from '../tokens';
+import { lightTheme as baseLightTheme, darkTheme as baseDarkTheme } from './theme.web';
+
+// For  styled components in React Native: converts unitless numbers to "px" strings
+const convertToPxString = (source) =>
```

Comment:
rename `convertToPxString` to `getPxValue`
rename `source` to `value`

Also this is a generic function so it should just accept a value and return the px value the reduce should not be part of this function